sol-did/client/packages/core/src/DidSolIdentifier.ts

```diff
@@ -104,7 +104,7 @@ export class DidSolIdentifier {
    * @param did the did string
    */
   static parse(did: string | VerificationMethod): DidSolIdentifier {
-    if (typeof did == 'string') {
+    if (typeof did === 'string') {
```

Comment:
Do we want to use the isStringDID guard here too?

Reply:
Probably makes sense. I did not want to change existing code (but obviously I did).